pkg/loadtest/client_kvstore.go

```diff
@@ -3,7 +3,7 @@ package loadtest
 import (
 	"fmt"
 
-	"github.com/tendermint/tendermint/libs/common"
```

Comment:
the tendermint/libs/common directory was split into multiple directories. Iavl should be imported into this repo.